### PR TITLE
Home explore page

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Added Home and Explore page to the video library with category filtering.
Please review and provide feedback to improve the code.

### CODE REVIEW

File: src/components/Navbar/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+.auth-btn{
+    padding: 10px 16px;
+    border-radius: 5px;
+    margin-right: 15px;
+    border:none;
+}
+.theme-icon{
+    font-size:24px;
+    color:white;
```
Comment (dhruvin25799) [inline src/components/Navbar/Navigation.css]: Suggestion : Use variable names for colors, IMO it makes it easy when debugging to see where you used which color.

File: src/components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  flex-direction: row;
+}
+.sidebar-icons {
+  padding-right: 20px;
+  height: 30px;
+  width: 20px;
+}
+
+.sidebar-links:hover {
+  background-color: #536075;
```
Comment (dhruvin25799) [inline src/components/Sidebar/Sidebar.css:24]: Same color variables here.
You're using #000000 for black, so you can just make a variable --COLOR-black

**Review by dhruvin25799 — APPROVED** [review]
Code looks clean. Good work using IIFEs, class names are good.
Good to merge, just added some suggestions for CSS.

File: src/backend/db/categories.js [hunk 3]
```diff
@@ -8,20 +8,33 @@ import { v4 as uuid } from "uuid";
...
-    categoryName: "Computer Programming",
+    category: "photography",
     description:
       "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
   },
   {
     _id: uuid(),
-    categoryName: "Frontend Development",
+    category: "communication",
     description:
```
Comment (ArpitPatni) [inline src/backend/db/categories.js]: Capitalize the first letter of the category names.It doesn't look good in the UI.Other than that code looks clean and good to merge.

File: src/backend/db/categories.js [hunk 4]
```diff
@@ -8,20 +8,33 @@ import { v4 as uuid } from "uuid";
...
       "Front-end web development, also known as client-side development is the practice of producing HTML, CSS and JavaScript for a website or Web Application",
   },
   {
     _id: uuid(),
-    categoryName: "Backend Development",
+    category: "video editing",
+    description:
```
Comment (ArpitPatni) [inline src/backend/db/categories.js]: Capitalize the first  letter of the catgeory names,other than that codes looks clean and good to merge.

**Pull request merged**